app/src/main/java/in/avimarine/boatangels/activities/AskInspectionActivity.java

```diff
@@ -0,0 +1,109 @@
+package in.avimarine.boatangels.activities;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import butterknife.OnClick;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.DocumentReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import in.avimarine.boatangels.R;
+import in.avimarine.boatangels.db.objects.Boat;
+import in.avimarine.boatangels.db.objects.User;
+
+public class AskInspectionActivity extends AppCompatActivity {
+
+  private Boat inspectBoat;
+  private User user;
+  private String getBoatUid;
+  private int yachtiePoint;
+  private FirebaseFirestore db = FirebaseFirestore.getInstance();
+  private String Uid;
+  private String points;
+  private final String  p = " points";
+
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_ask_for_inspection);
+    DocumentReference docRef = db.collection("users").document(FirebaseAuth.getInstance().getUid());
```

Comment:
Why don't you use the FireBase class?
Accessing the DB directly without using the class designed for it is bad practice.